Inc/Model.h

```diff
@@ -163,6 +225,44 @@ namespace DirectX
             bool wireframe = false,
             _In_opt_ std::function<void __cdecl()> setCustomState = nullptr) const;
 
+        // Draw all the meshes using model bones
+        void XM_CALLCONV Draw(
+            _In_ ID3D11DeviceContext* deviceContext,
+            const CommonStates& states,
+            size_t nbones, _In_reads_(nbones) const XMMATRIX* boneTransforms,
+            FXMMATRIX world, CXMMATRIX view, CXMMATRIX projection,
+            bool wireframe = false,
+            _In_opt_ std::function<void __cdecl()> setCustomState = nullptr) const;
+
+        // Draw all the meshes using skinning
+        void XM_CALLCONV DrawSkinned(
+            _In_ ID3D11DeviceContext* deviceContext,
+            const CommonStates& states,
+            size_t nbones, _In_reads_(nbones) const XMMATRIX* boneTransforms,
+            FXMMATRIX world, CXMMATRIX view, CXMMATRIX projection,
+            bool wireframe = false,
+            _In_opt_ std::function<void __cdecl()> setCustomState = nullptr) const;
+
+        // Compute bone positions based on heirarchy and transform matrices
```

Comment:
Comment and function name don't align in their use... Maybe the function should be `ComputeAbsoluteBoneTransformsTo` ?

Copy sounds like it's just copying, not doing computations AND copying...

Reply:
Or `CopyAndUpdate...`

Reply:
The name is taken from the *XNA Game Studio* methods which do the same thing.

<html>
<body>
<!--StartFragment-->

  | CopyAbsoluteBoneTransformsTo | Copies a transform of each bone in a model relative to all parent bones of the bone into a given array.
-- | -- | --
  | CopyBoneTransformsFrom | Copies an array of transforms into each bone in the model.
  | CopyBoneTransformsTo | Copies each bone transform relative only to the parent bone of the model to a given array.

<!--EndFragment-->
</body>
</html>